descriptions/ab+/ja.lua

```diff
@@ -14,7 +14,7 @@ local languageCode = "ja"
 EID.descriptions[languageCode] = {}
 EID.descriptions[languageCode].custom = {} -- table for custom entity descriptions
 EID.descriptions[languageCode].languageName = "Japanese"
-EID.descriptions[languageCode].alternativeLanguageCodes = {"ja_jp", "jp"}
+EID.descriptions[languageCode].alternativeLanguageCodes = {"japanese", "ja_jp"}
```

Comment:
"jp" needs to be supported as well, because thats the ingame Japanese language code